lib/frame.rb

```diff
@@ -0,0 +1,27 @@
+class Frame
+
```

Comment:
Classes are readable and give you a clear idea of the responsibility of each method (very easy to follow what would happen just by reading the code)